src/fhir-types.ts

```diff
@@ -160,6 +160,18 @@ export interface Reference {
   display?: string;
 }
 
+export type PublicationStatus = '	draft' | 'active' | 'retired' | 'unknown';
```

Comment:
Is this supposed to be `'	draft'` with the tab? I assume that's a copy-paste error from the FHIR specs.

Reply:
Yes this was a typo (from copy-pasting)! I have removed this in d3a48a3.